Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -647,6 +644,32 @@ function vtkResliceCursorWidget(publicAPI, model) {
       );
     }
   );
+
+  publicAPI.getPlaneExtremities = (viewType) => {
+    const dirProj = publicAPI.getWidgetState().getPlanes()[viewType].normal;
+    const length = vtkBoundingBox.getDiagonalLength(
+      publicAPI.getWidgetState().getImage().getBounds()
+    );
+    const p1 = vtkMath.multiplyAccumulate(
+      publicAPI.getWidgetState().getCenter(),
+      dirProj,
+      -length,
+      []
+    );
+    const p2 = vtkMath.multiplyAccumulate(
+      publicAPI.getWidgetState().getCenter(),
+      dirProj,
+      length,
+      []
+    );
+    // FIXME: support oriented bounds
+    const intersectionPoints = vtkBox.intersectWithLine(
```

Comment:
An idea of solution:
From the image direction, we already have the 3 normals of the 6 planes of the image oriented bounding box.
Now let's say we have for each plane `i` the equation `X.N_i = a_i` (the equation of a plane, we get `a_i` using indexToWorld)
And the equation of the line `X = b * N`
Then the intersection of the plane `i` and the line verifies: `(b * N). N_i = a_i` so `b = a / (N.N_i)` when `N.N_i` is not 0.
We get 6 different values for `b` and only keep the biggest negative and the smallest positive (if there is a negative and a positive value).
From `b`, we get the point by doing `X = b * N`.